org.eclipse.scanning.points/src/org/eclipse/scanning/points/ScanPointGeneratorFactory.java

```diff
@@ -65,6 +65,10 @@ public void run() {
         return new JythonObjectFactory<>(ScanPointIterator.class, "jython_spg_interface", "JArrayGenerator");
     }
 
+	public static JythonObjectFactory<ScanPointIterator> JStaticGeneratorFactory() {
+		return new JythonObjectFactory<>(ScanPointIterator.class, "jython_spg_interface", "JStaticPointGenerator");
+	}
+
 	public static JythonObjectFactory<ScanPointIterator> JSpiralGeneratorFactory() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)